cloudflare/fetch/client.go

```diff
@@ -0,0 +1,24 @@
+package fetch
+
+import (
+	"net/http"
+	"syscall/js"
+
+	"github.com/syumai/workers/internal/jsutil"
+)
+
+// Client is an HTTP client.
+type Client struct {
+	*http.Client
```

Comment:
Is the `*http.Client` embedded field needed? It does not appear to be used at this time.
If it is needed, I would like to know why.

Reply:
I was thinking we could support an API client that includes options like `WithHTTPClient`. However, it is not currently used. Fixed.